examples/CV2_quantum-neural-net.py

```diff
@@ -42,7 +42,7 @@ def quantum_neural_net(var, x=None):
         x (array[float]): single input vector
 
     Returns:
-        float: expectation of Homodyne measurement on Mode 0
+        float: expectation of QuadOperator measurement on Mode 0
```

Comment:
Looks like this was originally a typo. Should be expectation of X measurement

Reply:
Changed it